packages/react-router-node/tsconfig.json

```diff
@@ -10,7 +10,6 @@
     "strict": true,
     "jsx": "react",
     "declaration": true,
-    "emitDeclarationOnly": true,
```

Comment:
Can't have this with `noEmit` - I assume `tsup` is taking care of our emitting though?  @jacob-ebey can you confirm this won't mess with anything?